common/plugins/com.liferay.ide.core/plugin.xml

```diff
@@ -71,7 +71,16 @@
          point="org.eclipse.core.resources.natures">
       <runtime>
          <run
-               class="com.liferay.ide.core.LiferayNature">
+               class="com.liferay.ide.core.LiferayProjectNature">
+         </run>
+      </runtime>
+   </extension>
+   <extension
+         id="com.liferay.ide.core.liferayWorkspaceNature"
+         point="org.eclipse.core.resources.natures">
+      <runtime>
+         <run
+               class="com.liferay.ide.core.LiferayWorkspaceNature">
```

Comment:
Agreed, we do have some special actions for liferay workspace nature.  However, perhaps we can use simple tests to determine if the project is a workspace project or normal module.  Natures should only be added when there is no easy way to determine otherwise.  So i'd like to avoid adding a 2nd nature because both 1) and 2) can be accomplished without a nature and isn't very expensive.  (natures are used when determining the project type is expensive, like the case of maven or gradle module projects.)